src/xdg-autostart-generator/xdg-autostart-service.c

```diff
@@ -597,6 +597,7 @@ int xdg_autostart_service_generate_unit(
         fprintf(f,
                 "\n[Service]\n"
                 "Type=exec\n"
+                "ExitType=cgroup\n"
```

Comment:
It'd be nice to split this chunk to a separate commit… It's a separate logical change and it'll makes bisecting easier to have two commits.

Reply:
will